### PR TITLE
Feature header

### PR DESCRIPTION
**Pull request by retrotshepo**

### CODE REVIEW

File: src/containers/header/index.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+	
+	float: left;
+	margin: 10px;
+	
+	font-family: sans-serif;
+	color: white;
+	font-size: 17px;
+	
+	border: none;
+	}
```
Comment (phlippieb) [inline src/containers/header/index.css]: welp

File: src/containers/header/index.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    background-color: #FB6902;
+    border: 6px solid white;
+	
+	margin-left: 10px;
+	float: left;
+}
+
+#btn-square1{
+	width: 50px;
+    height: 50px;
```
Comment (phlippieb) [inline src/containers/header/index.css]: indentation

File: src/containers/header/index.css [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    background-color: #FB6902;
+    border: 6px solid white;
+	
+	margin-left: 10px;
+	float: left;
+}
+
+.account-square-2{
+	border-radius: 50%;
+	width: 50px;
```
Comment (phlippieb) [inline src/containers/header/index.css]: indentation

File: src/containers/header/index.css [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    height: 50px;
+	border-radius: 50%;
+	padding: 0px;
+	border: none;
+	margin: 1px;
+}
+#btn-square2{
+	width: 50px;
+    height: 50px;
```
Comment (phlippieb) [inline src/containers/header/index.css]: indentation

File: src/containers/header/index.css [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+.chatname{
+	font-family: sans-serif;
+	color: white;
+    float: left;
+	margin-right: 1px;
+}
+
+.account-square-1{
+	border-radius: 50%;
```
Comment (NardusVanDerVyver) [inline src/containers/header/index.css]: Indentation throughout the file needs to be consistent.

**Review by NardusVanDerVyver — CHANGES_REQUESTED** [review]

**Review by phlippieb — CHANGES_REQUESTED** [review]

File: src/containers/header/index.css (hunk 1 above)
Comment (phlippieb) [reply src/containers/header/index.css]: by which I mean, check the indentation of this bracket

File: src/containers/header/index.js [hunk 6]
```diff
@@ -4,9 +4,36 @@ import Login from '../login'
 import Register from '../register'
 import Body from '../body'
 
+import _profileimg1 from './ic_account_circle_black_48dp.png'
+import _profileimg2 from './ic_schedule_black_48dp.png'
+
+
```
Comment (bsibiya) [inline src/containers/header/index.js]: The imports must be grouped together

File: src/containers/header/index.css [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+	height: 50px;
+	background-color: white;
+	margin-left: 5px;
+	float: left;
+	border: none;
+	padding: 0px;
+}
+
+#buttonAbout{
+	border-radius: 50%;
```
Comment (jaderabbit) [inline src/containers/header/index.css]: why buttonAbout and buttonProfile? why not just a button?

**Review by jaderabbit — CHANGES_REQUESTED** [review]
One more thing

**Review by phlippieb — APPROVED** [review]

File: src/containers/header/index.css (hunk 7 above)
Comment (retrotshepo) [reply src/containers/header/index.css]: they have different values for "margin-left" but I will make them reference one. do you think I should do the same for "new chat" and "new group" buttons as well? Never mind Kimothy gave it an "okay"

**Review by jaderabbit — APPROVED** [review]

File: src/containers/header/index.css (hunk 3 above)
Comment (retrotshepo) [reply src/containers/header/index.css]: resolved, together with naming convention

**Review by phlippieb — APPROVED** [review]

**Review by NardusVanDerVyver — APPROVED** [review]

**Pull request merged**